pkg/proxy/ipvs/ipset.go

```diff
@@ -72,6 +72,11 @@ const (
 	kubeNodePortLocalSetSCTP        = "KUBE-NODE-PORT-LOCAL-SCTP"
 )
 
+// This is the list of IP Sets that need to be destroyed on start up for backwards compatbility reasons
+// If a proporty of an IP Set is changed and if kube-proxy is required to recreate it, then add the
+// set name into cleanupIPSetsOnStartup
+var cleanupIPSetsOnStartup = sets.NewString(kubeNodePortSetSCTP, kubeNodePortLocalSetSCTP)
```

Comment:
I don't like this pattern of white listing ipsets that need a clean up before starting but not sure what the alternatives are. We did this for proxy rules and that ended up being EOL'd. I was thinking of adding `ReplaceSet` in `utilipset.Interface` which does flush / destroy / create, but that would result in destroying and recreating the ipset on every proxy sync which I don't think is ideal.

Reply:
The alternatives are:
1) create a new ipset with a different name going forward
2) revert https://github.com/kubernetes/kubernetes/pull/74341 - I don't think this is an option if we want to support sctp

Reply:
cc @lbernail